Doc/whatsnew/3.14.rst

```diff
@@ -293,13 +293,19 @@ concurrent.futures
   same process) to Python code.  This is separate from the proposed API
   in :pep:`734`.
   (Contributed by Eric Snow in :gh:`124548`.)
+
 * The default ``ProcessPoolExecutor`` start method (see
   :ref:`multiprocessing-start-methods`) changed from *fork* to *forkserver* on
   platforms other than macOS & Windows. If you require the threading
   incompatible *fork* start method you must explicitly request it by
   supplying a *mp_context* to :class:`concurrent.futures.ProcessPoolExecutor`.
   (Contributed by Gregory P.  Smith in :gh:`84559`.)
 
+* Introduced the optional ``buffersize`` parameter to
+  :meth:`concurrent.futures.Executor.map`, which specifies the number
+  of results that can be buffered before being yielded.
+  (Contributed by Enzo Bonnal and Josh Rosenberg in :gh:`74028`.)
```

Comment:
```suggestion
* Add the optional ``buffersize`` parameter to
  :meth:`concurrent.futures.Executor.map` to specify the number
  of results that can be buffered before being yielded.
  (Contributed by Enzo Bonnal and Josh Rosenberg in :gh:`74028`.)
```

In general we sync the NEWS entry with the What's New entry.

Reply:
thanks!